libraries/chain/database.cpp

```diff
@@ -3901,9 +3900,15 @@ void database::update_virtual_supply()
          auto percent_sbd = uint16_t( ( ( fc::uint128_t( ( dgp.current_sbd_supply * get_feed_history().current_median_history ).amount.value ) * STEEM_100_PERCENT )
```

Comment:
It looks like this statement would be unnecessary after hardfork 21 and should probably go in an `else` statement.